tests/sles4sap/trento/test_hana_stop.pm

```diff
@@ -11,35 +11,65 @@ use base 'consoletest';
 use testapi;
 use serial_terminal 'select_serial_terminal';
 use utils 'script_retry';
-use qesapdeployment qw(qesap_upload_logs qesap_ansible_cmd);
+use qesapdeployment;
 use trento;
 
 
 sub run {
     my ($self) = @_;
     select_serial_terminal;
 
+    my $primary_host = 'vmhana01';
+
     my $prov = get_required_var('PUBLIC_CLOUD_PROVIDER');
-    qesap_ansible_cmd(cmd => 'crm status', provider => $prov, filter => 'vmhana01');
-    qesap_ansible_cmd(cmd => 'SAPHanaSR-showAttr', provider => $prov, filter => 'vmhana01');
+    qesap_ansible_cmd(
+        cmd => 'crm status',
+        provider => $prov,
+        filter => $primary_host);
+    qesap_ansible_cmd(cmd => 'SAPHanaSR-showAttr --format=script',
+        provider => $prov,
+        filter => $primary_host);
+
+    # Stop the primary DB
+    qesap_ansible_cmd(
+        cmd => "su - hdbadm -c 'HDB stop'",
+        provider => $prov,
+        filter => $primary_host);
 
-    qesap_ansible_cmd(cmd => "su - hdbadm -c 'HDB stop'", provider => $prov, filter => 'vmhana02');
+    my $_monitor_start_time = time();
+    my $done;
+    while ((time() - $_monitor_start_time <= 300) && (!$done)) {
+        sleep 30;
+        my $show_attr = qesap_ansible_script_output(
+            cmd => 'SAPHanaSR-showAttr',
+            provider => $prov,
+            host => $primary_host,
+            root => 1);
 
-    my $cmr_status = qesap_ansible_script_output(cmd => 'crm status', provider => $prov, host => 'vmhana01', root => 1);
-    record_info("crm status", $cmr_status);
-    qesap_ansible_cmd(cmd => 'SAPHanaSR-showAttr', provider => $prov, filter => 'vmhana01');
+        my %status = {};
```

Comment:
Did this work? IIRC `%status = {}` declares a reference to an empty hash instead of an empty hash:
```
%status = {};
use Data::Dumper;
print Dumper \%status;
$VAR1 = {
          'HASH(0x55f36390d220)' => undef
        };
```
Which means you would access elements there with `$status->{value}` instead of `$status{value}`.

Empty hashes on the other hand would be declared with `()`:
```
%status = ();
use Data::Dumper;
print Dumper \%status;
$VAR1 = {};
```
If it worked, no need to change it. I just found it odd.